src/sdx_pce/topology/temanager.py

```diff
@@ -37,6 +37,7 @@
 )
 
 UNUSED_VLAN = None
+MAX_OXP_DEFAULT = 4294967295
```

Comment:
This looks good to me! Just one quick question: why we have such a large MAX_OXP_DEFAULT? Where is this number coming from?

Reply:
Great question, Cong! I briefly mentioned in the description something about it:

> I introduced MAX_OXP_DEFAULT as a way to differentiate cases where max_number_oxps was not required (it would be 101 as well, no difference)

In summary, any value above 100 would be fine (e.g., 101).  Since we see over and over projects that estimate a reasonable growing throughout the time and expectations change, I decided to go with a higher number. `4294967295 = 0xffffffff` :)